galaxy_api/api/v3/viewsets.py

```diff
@@ -183,7 +183,7 @@ def get(self, request, *args, **kwargs):
 
         if response.status_code == requests.codes.ok:
             return StreamingHttpResponse(
-                response.content,
+                response.iter_content(),
```

Comment:
May make sense to use chunk_size=None here
``` python
response.iter_content(chunk_size=None)
```
 based on https://2.python-requests.org/en/master/api/#requests.Response.iter_content:

> chunk_size must be of type int or None. A value of None will function differently depending on the value of stream. stream=True will read data as it arrives in whatever size the chunks are received. If stream=False, data is returned as a single chunk.

Reply:
Yeah, definitely needs chunk_size=None or larger than one. The default chunk_size=1 causes small ish downloads
to be pretty slow. In an all localhost test, default chunk_size=1 for ~200kb to avg about 7 seconds and ~26kb/s:

```
Downloading to "alikins.k8s_demo_collection.0.37.3-1"
Done. 192.72 kB in 7.32279s (26.32 kB/s)
```

And chunk_size=None, downloads average about 0.001 seconds and 159MB/s (localhost in full effect...)

```
Downloading to "alikins.k8s_demo_collection.0.37.3"
Done. 192.72 kB in 0.00121s (154.99 MB/s)
```